### PR TITLE
[chore] Prepare release v0.76.0/1.0.0-rc10

### PR DESCRIPTION
**Pull request by opentelemetrybot**
The following commands were run to prepare this release:
- make chlog-update VERSION=v0.76.0/v1.0.0-rc10
- make prepare-release GH=none PREVIOUS_VERSION=1.0.0-rc9 RELEASE_CANDIDATE=1.0.0-rc10 MODSET=stable
- make prepare-release GH=none PREVIOUS_VERSION=0.75.0 RELEASE_CANDIDATE=0.76.0 MODSET=beta

### CODE REVIEW

**Review by dmitryax — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,38 @@
 
 <!-- next version -->
 
+## v1.0.0-rc10/vv0.76.0
```
Comment (djaglowski) [inline CHANGELOG.md:7]:
```suggestion
## v1.0.0-rc10/v0.76.0
```

Comment (djaglowski) [inline CHANGELOG.md:7]:
```suggestion
## v1.0.0-rc10/v0.76.0
```

Comment (djaglowski) [reply CHANGELOG.md:7]: For some reason I can't commit this suggestion.

**Comment by djaglowski** [comment]
I will rerun contrib tests once https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/21119 is merged

File: CHANGELOG.md (hunk 1 above)
Comment (codeboten) [reply CHANGELOG.md:7]: Hmmm...i also can't submit the suggestion :(

**Comment by djaglowski** [comment]
Closing in favor of #7565.

I reran the workflow with proper parameters to see if this would fix the problems in contrib tests. No luck, but it does contain at least one additional change in `cmd/builder/internal/builder/config.go`.

**Pull request closed without merging**